Bottom: 5825ed758170c4562e42a7e28a99c3e2b97d685b
Top:    d1cf78dc906ff3ca97939c6679ddd669e8c24037
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2008-10-22 00:30:01 +0100

Remove "=" shortcut to the djopt() thingy which keeps eating my design

ARR!! how much work have I lost to this key binding!






---

diff --git a/src/gpcb-menu.res b/src/gpcb-menu.res
index b09d948..93b6073 100644
--- a/src/gpcb-menu.res
+++ b/src/gpcb-menu.res
@@ -310,13 +310,11 @@ MainMenu =
    {"Rip up all auto-routed tracks" RipUp(All)}
    -
    {"Optimize routed tracks"
-    {"Auto-Optimize" djopt(auto)  a={"Shift-=" "Shift<Key>="}}
     {"Debumpify" djopt(debumpify) }
     {"Unjaggy" djopt(unjaggy) }
     {"Vianudge" djopt(vianudge) }
     {"Viatrim" djopt(viatrim) }
     {"Ortho pull" djopt(orthopull) }
-    {"Simple optimization" djopt(simple)  a={"=" "<Key>="}}
     {"Miter" djopt(miter) }
     {"Puller" a={"Y" "<Key>y"} Puller() }
     {"Global Puller"
